### DISCUSSION TITLE
Help! can't get a successful print

### DISCUSSION THREAD

**Discussion by larie438** [mriscoc/Ender3V2S1#372] [Q&A]
Since installing this firmware on my Ender 3V2 and using auto bed leveling, I cannot get a successful print, the material does not seem to stick to the plate, and I'm having massive warping and lifting issues. I was able to print fine before I activated mesh leveling in my Cure starting g code, I tried different g code variations, but none seems to be working can someone help?

here is my current starting g code:

; Ender 3 Custom Start G-code
M190 S{material_bed_temperature} ;Heat bed to Cura and wait
M104 S150 ;Heat hotend, no wait
G28 ; Home all axes
G29 ; BLtouch bed level
M500 ; save mesh enabled
G92 E0 ; Reset Extruder
G1 Z12 F3000 ;Set nozzle for heatup
G1 X0 Y0 ;Set nozzle to wipe
M109 S{material_print_temperature} ;Final hotend heat
G92 E0 ; Reset Extruder
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position
G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line
G1 X0.4 Y200.0 Z0.3 F5000.0 ; Move to side a little
G1 X0.4 Y20 Z0.3 F1500.0 E30 ; Draw the second line
G92 E0 ; Reset Extruder
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X5 Y20 Z0.3 F5000.0 ; Move over to prevent blob squish

THANKS!

**Comment by MaynardMann** [comment]
auto mesh leveling with the cr touch on s1 is currently very broken from what I have read and I also spent 3 days straight testing currently you're going to need to manually calibrate each point on the mesh manually with the edit mesh functions until it's right. I did it it took about 30 test prints and a few days using A 4x4 mesh but I decided its ultimately not worth it since it will need to be redone each time you tram your bed. I ordered a glass build plate to use till the software issues with auto bed leveling is fixed I haven't received it yet to test myself but any glass plate worth its salt should be flat enough that using the tram wizard in this software should be more than enough for great first layers.

**Reply by larie438** [reply]
I have a V2 not an s1

**Reply by ghost** [reply]
In my ender I always did automatic leveling on each print, but after a few days I started to do the leveling only once and read the recorded mesh for each print.
I'm not having a fixing problem. What I recommend is to adjust the nozzle distance.
I will post my gcode that I use in cura.

; Ender 3 V2 G-code inicial customizado 07_09_2022
M117 Aquecendo a mesa
M140 S{material_bed_temperature_layer_0} ; ajusta a temperatura da mesa
M105 ; Informa a temperatura para o octoprint
M190 S{material_bed_temperature_layer_0} ; aguarda a mesa chegar na temperatura final
G92 E0 ; Reseta extrusora
G28 ; Home todos os eixos
M420 S1 ; Faz a leitura da malha gravada na memória
M104 S{material_print_temperature_layer_0} ; ajusta a temperatura da extrusora
M105 ; Informa a temperatura para o octoprint
M109 S{material_print_temperature_layer_0} ; aguarda a extrusora chegar na temperatura final
M117 Posicionando a extrusora
G1 Z2.0 F3000 ; Move o eixo Z para cima para evitar arranhão na mesa aquecida
G1 X0.1 Y20 Z0.3 F5000.0 ; Move para a posição inicial
G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Desenha a primeira linha
G1 X0.4 Y200.0 Z0.28 F5000.0 ;Move um pouco para o lado
G1 X0.4 Y20 Z0.28 F1500.0 E30 ;Desenha a segunda linha
G92 E0 ;Reseta extrusora
G1 Z2.0 F3000 ;Move o eixo Z para cima para evitar arranhão na mesa aquecida
G1 X5 Y20 Z0.3 F5000.0 ; Mova-se para evitar o blob squish
M117 Imprimindo

**Reply by larie438** [reply]
Thanks but I'm not using octoprint right now, I'm planning on adding it soon, but right now I just would like to be able to have a successful print.
Can I just delete your octoprint lines or do I need to do something else?

**Reply by ghost** [reply]
> Obrigado, mas não estou usando octoprint no momento, estou planejando adicioná-lo em breve, mas agora só gostaria de poder ter uma impressão bem-sucedida. Posso apenas excluir suas linhas octoprint ou preciso fazer outra coisa?

you can use the whole gcode even not using octoprint, insert this gcode in cura

**Reply by curather** [reply]
I had a issue when I first had my V2 where I would constantly have to do a auto mesh.  The problem I found on my unit was that a corner was bent.  I had to add aluminum foil to the front right corner until the gap between the metal heating plate and the glass bed was gone.  This fixed the constant changes to my bed level.
